spec/controllers/api/sequences/sequences_create_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+describe Api::SequencesController do
+
+  include Devise::TestHelpers
+
+  describe '#create' do
+
+    let(:user) { FactoryGirl.create(:user) }
+
+    it 'creates a new sequences for a user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.